changelog/unreleased/bug-fixes/1804.md

```diff
@@ -0,0 +1,2 @@
+The disk budget feature no longer triggers are rare segfault while deleting
```

Comment:
`s/are/a`?

Reply:
I guess it was already to late in the evening when I wrote that. I amended and force pushed the commit.